suite/tests/api/ir_aarch64.c

```diff
@@ -3960,6 +3960,59 @@ test_asimddiff(void *dc)
     test_instr_encoding(dc, OP_umull2, instr);
 }
 
+static void
+test_sys(void *dc)
+{
+    instr_t *instr;
+
+    /* SYS #<op1>, <Cn>, <Cm>, #<op2>{, <Xt>}
+     *
+     * Data cache operations are aliases of SYS:
+     * DC <dc_op>, <Xt>
+     * is equivalent to
+     * SYS #<op1>, C7, <Cm>, #<op2>, <Xt>
+     */
+
+    /* DC ZVA, Xt => SYS #3, C7, C4, #1, Xt */
+    instr = INSTR_CREATE_sys(
+        dc, opnd_create_base_disp_aarch64(DR_REG_X0, DR_REG_NULL, 0, false, 0, 0, OPSZ_0),
```

Comment:
So this OPSZ_0 would become the named constant from above.

Reply:
Done in latest commit https://github.com/DynamoRIO/dynamorio/pull/4386/commits/cd48ffae460dddcad2e004c28ceabff542b7342c